### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by inductor**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
N/A

**Details of Changes**
Add details of changes here.
- Update Japanese translation

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@inductor before we merge this, can you add your agreement to #7625?

**Comment by inductor** [comment]
@yair100 Done, thanks.

**Review by yair100 — CHANGES_REQUESTED** [review]

File: src/Files/MultilingualResources/Files.ja-JP.xlf [hunk 1]
```diff
@@ -3247,19 +3247,19 @@ App Centerを使用して、アプリの使用状況を追跡し、バグを見
         </trans-unit>
         <trans-unit id="ChooseCustomIcon" translate="yes" xml:space="preserve">
           <source>Choose a custom folder icon</source>
-          <target state="new">Choose a custom folder icon</target>
+          <target state="new">下の一覧からアイコンを選択</target>
```
[the remark this answers is not in the document]
Comment (inductor) [reply src/Files/MultilingualResources/Files.ja-JP.xlf]: @yair100 addressed

**Review by yair100 — APPROVED** [review]
LGTM

**Pull request merged**